business_requirement_deliverable/models/business.py

```diff
@@ -78,8 +78,6 @@ def product_id_change(self):
             uom_id = product.uom_id.id
         if product.description_sale:
             description += '\n' + product.description_sale
-        if not self.name:
```

Comment:
@rubencabrera 
After reconsidering: why should we have this change in this PR?
I think you need to separate this modification that was done in another PR (may rebase)?
@victormartinelicocorp @darshan-serpent